### PR TITLE
Fixes holy water not blocking jaunt

### PR DESCRIPTION
**Pull request by Pathid**
:cl:
* bugfix: Fixed the jaunt-blocking property of holy water. Also, since holy tiles block jaunt now, the chapel won't be possible to jaunt through.

### CODE REVIEW

**Comment by Pathid** [comment]
We might want to make it 5u to bless a turf though. It's hard for me to tell just how much this is going to nerf vamp/wiz/cult.

**Comment by iFlashYou** [comment]
Tiles outside of the chapel should be reblessed every 10 min to not have every round a jaunt free station

**Comment by Pathid** [comment]
One thing to keep in mind is that a fire extinguisher doesn't seem to work for blessing or holying turfs. I'm not sure what causes it since it works for lube.

Anyway, blood unholys tiles at 5u. Maybe it should remove the anti-jaunt at 1u to keep things symmetrical. Have to make sure it doesn't work on the mapped in NOJAUNT tiles though.

**Comment by ghost** [comment]
Kurfurst has the splashy stuff at https://github.com/vgstation-coders/vgstation13/issues/17434#issuecomment-365693921 Apparently it was already 5u.

So did a quick run by on local in case it helps with anything,
Holy water:
Container 5u splash blesses

Miniextinguisher 5u spray, doesn't bless even when you corner shot all 3 sprays
Extinguisher 5u spray doesn't bless
Spray bottle 5u, doesn't bless
Spray bottle 10u doesn't bless
Chemsprayer 5u blesses
Chemsprayer 10u blesses

Holy tiles and blessing turfs added in at https://github.com/vgstation-coders/vgstation13/pull/1834


Blood:
Container 5u splash defiles blood decal

Miniextinguisher 5u spray, doesn't defile, no blood decal
Extinguisher 5u spray doesn't defile, no blood decal
Spray bottle 5u, doesn't defile no blood decal
Spray bottle 10u, doesn't defile, blood decal
Chemsprayer 5u, defiles, blood decal
Chemsprayer 10u, defiles, blood decal

Beating a monkeyman, making drips doesn't defile
Beating a monkeyman, making a full blood decal doesn't defile
Dragging a beaten monkeyman making puddles of blood decal doesn't defile

Blood defilement is in chemistry_reagents.dm https://github.com/vgstation-coders/vgstation13/blob/Bleeding-Edge/code/modules/reagents/Chemistry-Reagents.dm
Added in at https://github.com/vgstation-coders/vgstation13/commit/e0bc7fc1cedf18a2df9447afc1f377585e5c5861

	if(volume >= 5 && !istype(T.loc, /area/chapel)) //Blood desanctifies non-chapel tiles
		T.holy = 0


Turned myself into a vampire, new holy tiles still fucked you up, defiling still fixed them.

tl;dr everything is retarded and nobody knew because nobody has VV and vampstended/worksonmylocalmachine.mp3/doesn'tworkonmylocalmachine.mp3 placebo cause we all got sick from blessed maint while seeing chaplains run around extinguishering everything

@iFlashYou Presumably if extinguishers were fixed so would the ways to defile it.  Hose it back down with blood then space cleaner. Or just leave a bunch of blood as bait for a grenade to kill him when someone reports it.

**Comment by Pathid** [comment]
> Apparently it was already 5u.

Yeah, `holy` (which does smiting) was set to "on" at 5u. And I didn't change that. I was thinking along the lines that, since `bless()` for objects is at 1u, `bless()` (which is what sets `NOJAUNT`) for turfs should also be 1u. So that's what I did. But I'm starting to believe having two different holy water statuses for turfs is very probably retarded from any point of view. There's no reason to have that complexity.

**Comment by Pathid** [comment]
This is what I should have done before. `NOJAUNT` is kept for mapping purposes, but `holy` is the only thing that's changed by holy water and blood. Jaunting checks for both now.

**Comment by Pathid** [comment]
@SarahJohnson489 Although there is a side effect I should mention from this latest commit. Chapel tiles are (almost) always holy, and so you will not be able to jaunt through the chapel (unless Nar-Sie comes and defiles it). This is consistent with the current behavior for astral journeying cultists and visible ghosts though, which were checking for `holy` all along.
